desktop/apps/show/components/artwork_item_metaphysics/save_controls/view.coffee

```diff
@@ -32,7 +32,6 @@ module.exports = class SaveControls extends Backbone.View
       mediator.trigger 'open:auth',
         mode: 'register'
         copy: 'Sign up to save artworks'
-        redirectTo: "#{@artwork.href}/save"
```

Comment:
These are removed since the server-side `/save` route was never implemented. Since we're no longer passing in `redirectTo`, the auth modal triggered when saving an artwork will take you to personalize, and then just bring you back where you started.

So...progress! The only thing left would be to make sure you actually save the artwork at some point.